RELEASE_NOTES.md

```diff
@@ -8,6 +8,21 @@ redirect_from:
 ---
 # Version history
 
+## Version 0.8.1
+
+DATE-OF-RELEASE: 🧑‍🤝‍🧑 The 18th draft changes to community governance model.
+
+* The project has switched to a community governance model.
+  * Governance model now covers steering team, composition, responsibilities, meetings, decision making process, lazy consensus, voting, and code of Conduct.
+  * The website is relocated to a new domain [https://standard.publiccode.net/](https://www.standardforpubliccode.org/).
```

Comment:
```suggestion
  * The website is now located at the new domain [https://standard.publiccode.net/](https://www.standardforpubliccode.org/).
```

Reply:
Adjusted the website url to the new one.